cmd/plugin-manager/list/list.go

```diff
@@ -96,14 +96,20 @@ func addFlagsForOptions(o *Flags, cmd *cobra.Command) {
 
 func (o *Options) run() error {
 	log := o.globalFlags.GetLogger()
-
+	o.PluginDir = fmt.Sprintf("%v/%v", o.PluginDir, plugin.MANAGED_DIR)
 	if o.Installed {
 		// retrieve list of all the plugins that are installed within plugin dir
+		// TODO: differentiate between multiple repos
 		plugins, err := plugin.GetFilteredPlugins(o.PluginDir, []string{}, log)
 		if err != nil {
 			return err
 		}
-		fmt.Println(fmt.Sprintf("Listing plugins from path - %s", o.PluginDir))
+		if len(plugins) > 1 {
```

Comment:
I think this might be better with the original. Otherwise if we add/remove default plugins, this language changes. How about just "Listing configured plugins from path", that way "configured" can mean the plugins found here and any defaults not needed to be installed.